### PR TITLE
feat: updated moses source package URL

### PR DESCRIPTION
**Pull request by Ashish13898**
This PR contains changes related to the modification of the source URL of Moses package in requirements.txt.

### CODE REVIEW

File: guacamol_baselines/moses_baselines/organ_train.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from moses.utils import CharVocab
 from multiprocessing import Pool
 
-from .common import read_smiles
+from common import read_smiles
```
Comment (drugilsberg) [inline guacamol_baselines/moses_baselines/organ_train.py:11]: I would keep the .common

File: dockers/requirements.txt [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 tqdm==4.26.0
 cython==0.29
 nltk==3.4.5
 flake8==3.5.0
-molsets @ git+https://github.com/molecularsets/moses.git@7b8f83b21a9b7ded493349ec8ef292384ce2bb52
\ No newline at end of file
+tensorflow
+keras
+h5py
+moses @ git+https://github.com/PaccMann/moses.git@0.0.1
```
Comment (drugilsberg) [inline dockers/requirements.txt:14]: Would it make sense to move Moses to gt4sd org first?

File: setup.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 import shutil
 
 SETUP_DIR = os.path.dirname(os.path.abspath(__file__))
+REQUIREMENTS = ["guacamol", "matplotlib", "torch", "joblib", "numpy", "tqdm", "cython", "nltk", "flake8"]
```
Comment (drugilsberg) [inline setup.py:14]: Do we really need flake8 for the install? Maybe we can remove it since I assume it's only for dev

**Review by drugilsberg — CHANGES_REQUESTED** [review]

File: dockers/requirements.txt (hunk 2 above)
Comment (Ashish13898) [reply dockers/requirements.txt:14]: I have just raised a new PR in the GT4SD org for Moses :)

Comment (Ashish13898) [reply dockers/requirements.txt:14]: Should we point to Moses of GT4SD org in this requirements as most likely we'll archive the Moses repo here in Paccman?

**Review by drugilsberg — APPROVED** [review]

**Pull request merged**